R/importSNV.TCGA.R

```diff
@@ -0,0 +1,156 @@
+# importNet.TCGA.R
+
+#' Import MAF files from source and coverts them to rSNV files.
+#'
+#' \code{importSNV.TCGA} imports maf, creates rSNV, returns rSNV
+#'
+#' @section MAF file:...
+#'
+#' @param fMAF vector of MAF file names
+#' @param rSNV new rSNV file name to be created or the one to be merged with
+#' @param silent Controls whether output to console should be suppressed. FALSE
+#'   by default.
+#' @param noLog Controls whether writing the result to the global logfile is
+#'   enabled. TRUE by default.
+#' @return rSNV file name containg data from all MAF files
+#'
+#' @family
+#'
+#'   ## @seealso \code{\link{fastMap}} fastMap() is used internally to map ENSP
+#'   IDs to gene symbols.
+#'
+#'   ## @examples ## \dontrun{ ## importNet.STRING(IN, OUT) ## }
+#' @export
+importSNV.TCGA <- function(fMAF,
+                             rSNV,
+                             silent = FALSE,
+                             noLog = TRUE) {
+
+    # ==== PARAMETERS ==========================================================
+
+    # NL <- .PlatformLineBreak()
+    # cutoffTypes <- c("xS", "xQ", "xN")
+    # defaultValues <- c(xS = 950, xQ = 0.9, xN = 10000)
+    #
+    # if (missing(val)) {
+    #     val <- defaultValues[cutoffType]
+    # }
+    #
+    # # Read header and one contents line
+    # tmp <- readLines(fName, n = 2)
+    #
+    # # Parse for requested column. STRING data is " "-delimited.
+    # header <- unlist(strsplit(tmp[1], " "))
+    # iCol <- which(net == header)  # column that contains the requested net
+    #
+    # data   <- unlist(strsplit(tmp[2], " "))
+
+
+    # ==== VALIDATIONS =========================================================
+
+    # General parameter checks
+    cR <- character()
+    cR <- c(cR, .checkArgs(fMAF,        like = c("FILE_E"),   checkSize = TRUE))
```

Comment:
This will fail if the input is a vector. You can do:
```
cR <- c(cR, .checkArgs(fMAF,
                    like = rep("FILE_E", length(fMAF)),
                    checkSize = TRUE))
```